### ISSUE TITLE
Apply new styles usin styled-components

### ISSUE THREAD

**Issue by Tzelon** [wmira/react-sidenav#20]
Hey,
I'm trying to apply new styles to the NavIcon element 
`import styled from 'styled-components';

const StyledNavIcon = styled(NavIcon)`
  width: 30px;
`;`

But when I use the new element I'm unable to see the child elements 
` <StyledNavIcon><FontAwesome name="th" size="x2" /></StyledNavIcon>`

Any insight ? 

Thank you

**Comment by wmira** [comment]
Hi @Tzelon ,
I miss to add this on the docs but the NavIcon and NavText are tagger elements. Meaning the ones that are rendered are their children. If you want to style the NavIcon then just add it on the child. The reason being so that its possible to reposition them elsewhere depending on the renderer (in future versions).

You can do the following:

```javascript
const IconCont = styled.div`
width: 30px;
display: inline-block;
`;

<NavIcon><IconCont><FontAwesome name="th" size="x2"/></IconCont></NavIcon>

```

**Comment by Tzelon** [comment]
Thank you !

**Comment by Tzelon** [comment]
Hey,
I'm still having an issue, I see that you use NavIconCont to wrap the Icon element.
NavIconCont have styles that I want to overwrite like width. Is there an easy way to do that ?

**Comment by wmira** [comment]
@Tzelon I have published 2.1.1 where NavIcon and NavText now supports overriding style and className
<NavIcon style={{yourstyle}} className='yourclassName'></NavIcon

**Comment by Tzelon** [comment]
Thanks works great
